### ISSUE TITLE
Segmentation Fault error at prepare PSFs step

### ISSUE THREAD

**Issue by blaisekuo** [Stuermer/EchelleSimulator#18]
Hi Julian, I ran EchelleSimiulator on a spectrograph model I generated and I got a Segmentation Fault at the  "Prepare PSFs" step. Any clues as to why? The HDF model file looks like it has some PSFs stored in it. I attached the model creation script I used.
[model_creation_iranti.txt](https://github.com/Stuermer/EchelleSimulator/files/4556380/model_creation_iranti.txt)

Running my EchelleSimulator build on the MaroonX example seems to be working.

**Comment by blaisekuo** [comment]
Ok, I've gotten past the error now by leaving just one config in the zemax file.

**Comment by Stuermer** [comment]
OK. I was just gonna write to you to send me the produced HDF file, maybe I can spot an error.  Let me know if you need anything else.

**Comment by blaisekuo** [comment]
Here's an HDF file I created. I am able to create the file and I can see that there are PSFs in the hdf file but when running it through EchelleSimulator I get a blank frame. I also ran the model_creation script on the zemax file provided and it also gives a blank frame.

I ran this on the command line:

./echellesimulator --spectrograph Iranti --constant 0.0001  -o ../simulations/flat.fit

[Iranti.zip](https://github.com/Stuermer/EchelleSimulator/files/4578856/Iranti.zip)

Looking in the hdf file, it looks like my problem is happening in the affine transformation calculation. Most of those values are missing in the order table.
